commit 093cfbf2b293b1215321a5de293013abd1d100f1
Author: Narrat
Date:   Mon Oct 24 20:15:24 2016 +0200

    ffmpeg: Enable mmal
    
    So mpv can use this to use the GPU on the pi
    
    Changes involved:
     * raspberry-firmware-tools to deps (needed for mmal)
     * change openjpeg to openjpeg2
       * not building in clean chroot and jpeg2 gets used
     * add libxv
       * another one caused by not building in a clean chroot

diff --git a/extra/ffmpeg/PKGBUILD b/extra/ffmpeg/PKGBUILD
index 746b0d9..9ac83e7 100644
--- a/extra/ffmpeg/PKGBUILD
+++ b/extra/ffmpeg/PKGBUILD
@@ -18,11 +18,11 @@ url='http://ffmpeg.org/'
 license=('GPL3')
 depends=('alsa-lib' 'bzip2' 'fontconfig' 'fribidi' 'gmp' 'gnutls' 'gsm' 'jack'
          'lame' 'libass' 'libavc1394' 'libbluray' 'libiec61883' 'libmodplug'
-         'libpulse' 'libsoxr' 'libssh' 'libtheora' 'libva' 'libvdpau' 'libwebp'
-         'opencore-amr' 'openjpeg' 'opus' 'schroedinger' 'sdl' 'speex'
+         'libpulse' 'libsoxr' 'libssh' 'libtheora' 'libva' 'libvdpau' 'libwebp' 'libxv'
+         'opencore-amr' 'openjpeg2' 'opus' 'schroedinger' 'sdl' 'speex'
          'v4l-utils' 'xvidcore' 'zlib'
          'libvidstab.so' 'libvorbis.so' 'libvorbisenc.so' 'libvpx.so'
-         'libx264.so' 'libx265.so' 'libnetcdf.so')
+         'libx264.so' 'libx265.so' 'libnetcdf.so' 'raspberrypi-firmware-tools')
 makedepends=('hardening-wrapper' 'ladspa' 'libvdpau' 'yasm')
 optdepends=('ladspa: LADSPA filters')
 provides=('libavcodec.so' 'libavdevice.so' 'libavfilter.so' 'libavformat.so'
@@ -33,6 +33,7 @@ validpgpkeys=('FCF986EA15E6E293A5644F10B4322F04D67658D8')
 sha256sums=('49cc3105f7891c5637f8fabb1b75ebb19c9b5123b311a3ccc6182aa35d58b89a'
             'SKIP')
 
+
 build() {
   cd ffmpeg-${pkgver}
 
@@ -80,6 +81,7 @@ build() {
     --enable-shared \
     --enable-version3 \
     --enable-x11grab \
+    --enable-mmal \
     $CONFIG
 
   make
